capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -69,9 +72,83 @@ export class SearchUserModal extends Component {
     })
   }
 
+  componentDidMount() {
+    this.fetchCollaborators();
+  }
+
+  fetchCollaborators = async () => {
+
+    if (!(this.props.type === 'booklists' && this.props.bookList)) {
+      return;
+    }
+
+    else if (this.props.bookList.collaboratorsIDs.length === 0) {
+      return;
+    }
+
+    let collaborators = [];
+
+    await Promise.all(this.props.bookList.collaboratorsIDs.map(async (collaboratorId) => {
+
+      const collaborator = await fetch(`/api/user?id=${collaboratorId}`).then(resp => resp.json());
+
+      delete collaborator.tokenObj;
+      collaborators.push(collaborator);
+
+    }));
+
+    // Checks for owner
+    collaborators = collaborators.filter((collaborator) => collaborator.id !== this.props.bookList.id);
```

Comment:
Not sure I follow this line here - are you trying to filter out the owner from the list of collaborators? Why do we compare `collaborator.id` to `this.props.bookList.id`?

Reply:
This line of code basically filter out any item which is doesn't return true based on the conditional. This means that it will remove the owner from the ```collaborators``` array. 

```this.props.bookList.id``` is actually suppose to be ```this.props.bookList.userID```

Reply:
Got it, thanks for the clarification!